kokoro/checks/clang_format/check.sh

```diff
@@ -16,17 +18,16 @@ if [ "$0" == "$SCRIPT" ]; then
   echo -e "> but changes outside of the scope of your PR won't affect the outcome"
   echo -e "> of this presubmit check.\n"
   while read line; do
-    clang-format --output-replacements-xml $line | grep '<replacement ' > /dev/null
-    if [ $? -eq 0 ]; then
+    if clang-format --output-replacements-xml $line | grep '<replacement ' > /dev/null; then
       echo $line
     fi
-  done <<< $(find "$DIR" -name '*.cpp' -o -name '*.h' \
+  done <<< $(find /mnt -name '*.cpp' -o -name '*.h' \
         | grep -v "third_party/" \
         | grep -v "/build" )
   echo -e "--\n"
 
-  cd "$DIR"
-  REFERENCE="origin/master"
+  cd /mnt
+  REFERENCE="${KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH:-origin/master}"
```

Comment:
Why does the REFERENCE variable here still contain the origin/master name? shouldn't that be set from outside?

Reply:
It's a fallback when "KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH" is not set. (Bash syntax ${name:-fallback_string}")

Reply:
This allows the script to be called on a standard linux machine for testing purposes. We can change that later to "main" without any pressure.

Reply:
Okay, thanks!

Reply:
Can you maybe add a comment there?

Reply:
Sure. About what exactly? THe master/main thing? Or about the default string? Regarding the latter I'm a bit hesitant because I feel it's a standard language construct widely used for that purpose and pretty clear on its own.

Reply:
If it is that widely used, then feel free to not add a comment.
Otherwise, I was referring to the default string. Something like "Use orgin/master as reference branch, if not specified otherwise."

Reply:
Yeah. I can live with that! :sweat_smile: Actually not too bad. Thanks!

Reply:
Done.